source/simulator/core.cc

```diff
@@ -1365,6 +1403,95 @@ namespace aspect
 
 
   template <int dim>
+  class VectorFunctionFromVectorFunctionObject : public Function<dim>
```

Comment:
@jdannberg Agreed, the description in the constructor should be moved to the class documentation.

@bangerth Not quite. There only is ```VectorFunctionFromScalarFunctionObject``` and ``` VectorFunctionFromTensorFunction```.